docs/DeveloperGuide.md

```diff
@@ -206,6 +206,56 @@ Step 4. The `Ui` calls `UiCommand#execute(MainWindow)`, which will result in a c
     * Cons: 
         * `Ui` and `CommandResult` are not closed to modification. A new instruction might require the addition of fields to `CommandResult` (to store instructions and related data) as well as a new conditional statement in `Ui` to handle the new instruction.
 
+### Update Commands [Coming soon in v1.3]
+
+CoLAB has several update commands for projects, events, deadlines, tasks and groupmates. They are used to edit details of entities that have already been created.
+
+The update project command and the update contact command are relatively straight forward. Below is a sequence diagram of how an update projecct command is executed.
```

Comment:
typo in projecct.

Also can consider styling it. `UpdateProject` command is executed